contracts/DCAStrategies/DCAStrategies/DCAStrategiesManagementHandler.sol

```diff
@@ -51,7 +51,24 @@ abstract contract DCAStrategiesManagementHandler is IDCAStrategiesManagementHand
   }
 
   /// @inheritdoc IDCAStrategiesManagementHandler
-  function updateStrategyTokens(uint80 _strategyId, IDCAStrategies.ShareOfToken[] memory _tokens) external override {}
+  function updateStrategyTokens(uint80 _strategyId, IDCAStrategies.ShareOfToken[] memory _tokens) external {
+    StrategyOwnerAndVersion memory _strategy = _strategies[_strategyId];
+    if (msg.sender != _strategy.owner) revert OnlyStratOwner();
+    if (_checkTokenSharesSanity(_tokens) == false) revert InvalidTokenShares();
+
+    uint16 _newVersion = ++_strategy.latestVersion;
```

Comment:
We can avoid modifying the value in memory
```suggestion
    uint16 _newVersion = _strategy.latestVersion + 1;
```

Reply:
But when I save the new version number? saving the entire struct at the end of the function?

Reply:
You are already saving it in line 60, with `_strategies[_strategyId].latestVersion = _newVersion;`

In line 59 you are increasing the version in the memory version of the struct, which is not necessary

Reply:
Got it, I understand now